Week5/PYTHON SOLUTIONWEEK-5.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "6f943092",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "281810c0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a=['sid','sname','city','contact']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "122adb31",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "records=[[1,'parrth','surat',9873215754],\n",
+    "         [2,'dhrumin','mahuva',9487521889],\n",
+    "         [3,'sunil','valod',3547985476],\n",
+    "         [4,'jenil','bardoli',6875489675],\n",
+    "         [5,'meet','baroda',9548759642]]\n",
+    "p=[]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "263f7b45",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "enter student id:6\n",
+      "enter student name:om\n",
+      "enter student city:bardoli\n",
+      "enter student contact number:6854796587\n",
+      "enter student id:7\n",
+      "enter student name:sai\n",
+      "enter student city:surat\n",
+      "enter student contact number:6954658759\n",
+      "enter student id:8\n",
+      "enter student name:ram\n",
+      "enter student city:mahuva\n",
+      "enter student contact number:9854769854\n",
+      "enter student id:9\n",
+      "enter student name:kishan\n",
+      "enter student city:valod\n",
+      "enter student contact number:9854457898\n",
+      "enter student id:10\n",
+      "enter student name:radha\n",
+      "enter student city:surat\n",
+      "enter student contact number:9876658712\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in range(5):\n",
+    "    sid=int(input(\"enter student id:\"))\n",
+    "    sname=input(\"enter student name:\")\n",
+    "    city=input(\"enter student city:\")\n",
+    "    contact=int(input(\"enter student contact number:\"))\n",
+    "    l=[sid,sname,city,contact]\n",
+    "    p.append(l)"
```

Comment:
why we use apeend ?

Reply:
append() method takes an object as an argument and adds it to the end of an existing list.

For example, suppose you create a list and you want to add another number to it.